src/lib/db/addon-store.ts

```diff
@@ -0,0 +1,132 @@
+import { Knex } from 'knex';
+import EventEmitter from 'events';
+import { Logger, LogProvider } from '../logger';
+import { IAddon, IAddonDto, IAddonStore } from '../types/stores/addon-store';
+
+import metricsHelper from '../util/metrics-helper';
+import { DB_TIME } from '../metric-events';
+import NotFoundError from '../error/notfound-error';
+
+const COLUMNS = [
+    'id',
+    'provider',
+    'enabled',
+    'description',
+    'parameters',
+    'events',
+];
+const TABLE = 'addons';
+
+export default class AddonStore implements IAddonStore {
+    db: Knex;
```

Comment:
Why are these public and not private?

Reply:
You mean the db field? just an oversight